lib/matplotlib/_mathtext.py

```diff
@@ -1865,6 +1865,8 @@ def csnames(group, names):
                 "|".join(map(re.escape, tex2uni)))
         )("sym").leaveWhitespace()
         p.unknown_symbol = Regex(r"\\[A-Za-z]+")("name")
+        def unclosed_group(self, s, loc, toks):
+            raise ParseFatalException(s, len(s), "Expected '}'")
```

Comment:
The location of this method is not that logical. I'd suggest moving it down to other group-methods. Otherwise, it looks good!

Reply:
Let me rephrase that: I think this should be removed as there is another method below (at a different level, so I wonder if this is used at all?).

Reply:
I think the method is being utilized because we get the same error message as above. 

Re: _there is another method below (at a different level_ - I might be missing something, could you point out which one?